consensus/src/round_manager.rs

```diff
@@ -867,4 +884,25 @@ impl RoundManager {
             .round(self.round_state.current_round())
             .epoch(self.epoch_state.epoch)
     }
+
+    /// Given R1 <- B2 if R1 has the reconfiguration txn, we inject error on B2 if R1.round + 1 = B2.round
+    /// Direct suffix is checked by parent.has_reconfiguration && !parent.parent.has_reconfiguration
+    ///
+    /// It's only enabled with fault injection (failpoints feature).
+    #[allow(unused_variables)]
+    fn should_inject_reconfiguration_error(block_data: &BlockData) -> bool {
+        #[cfg(feature = "failpoints")]
```

Comment:
why not tweak this a little bit and put the failpoints cfg code followed by the function that does this or something....